CalibTracker/SiStripHitEfficiency/src/HitEff.cc

```diff
@@ -81,6 +79,16 @@ HitEff::HitEff(const edm::ParameterSet& conf)
           consumes<edmNew::DetSetVector<SiStripCluster> >(conf.getParameter<edm::InputTag>("siStripClusters"))),
       digis_token_(consumes<DetIdCollection>(conf.getParameter<edm::InputTag>("siStripDigis"))),
       trackerEvent_token_(consumes<MeasurementTrackerEvent>(conf.getParameter<edm::InputTag>("trackerEvent"))),
+      topoToken_(esConsumes<TrackerTopology, TrackerTopologyRcd>()),
```

Comment:
For future reference, one can now just write
```C++
topoToken_(esConsumes());
```

Reply:
thanks @Dr15Jones, much more compact indeed. 
Applied here, wherever relevant.